f8a_worker/workers/bookkeeper.py

```diff
@@ -77,5 +77,5 @@ def execute(self, arguments):
             self.store_user_node(arguments, aggregated)
 
         postgres = StoragePool.get_connected_storage('BayesianPostgres')
-        postgres.store_api_requests(arguments.get('external_request_id'),
-                                    arguments.get('data'), aggregated)
+        if arguments['data'].get('api_name') == 'component_analyses_post':
```

Comment:
Why is this check required?

Reply:
@prashbnair I have kept the check as there is a check for sa bookkeeping as well.It will also help when this goes into stage.As otherwise other bookkeeping tasks will trigger this one..